<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des clients</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>1141</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Client">
     <attribute name="title">
      <string>Client</string>
     </attribute>
     <widget class="QGroupBox" name="groupBoxAjouterC">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>331</width>
        <height>401</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter client</string>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Identifiant du client :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditId">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>81</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Quantité d'articles :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditStatut">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>140</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditQuantitearticles">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>90</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Statut :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Produit acheté :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditProduitAchete">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>190</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAjouter">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>330</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAnnuler">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>330</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxConsulterC">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>30</y>
        <width>721</width>
        <height>421</height>
       </rect>
      </property>
      <property name="title">
       <string>Consulter client</string>
      </property>
      <widget class="QTableWidget" name="tableWidgetClient">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>601</width>
         <height>251</height>
        </rect>
       </property>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Idnetifiant du client</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Quantité d'articles</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Statut</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Produit acheté</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="pushButtonRechercher">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonModifier">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>380</y>
         <width>141</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier </string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonSupprimer">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>380</y>
         <width>141</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer </string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxExcel">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>80</y>
         <width>83</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>EXCEL</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxPDF">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>50</y>
         <width>83</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonExport">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>20</y>
         <width>70</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_Rechercher">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>30</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButtonRetour">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>460</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonMenuPrincipal">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>460</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Menu Principal</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Commande">
     <attribute name="title">
      <string>Commande</string>
     </attribute>
     <widget class="QGroupBox" name="groupBoxConsulterC_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>20</y>
        <width>681</width>
        <height>421</height>
       </rect>
      </property>
      <property name="title">
       <string>Consulter commande</string>
      </property>
      <widget class="QTableWidget" name="tableWidgetClient_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>721</width>
         <height>251</height>
        </rect>
       </property>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Identidiant du commande</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Quantité</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Réfférence</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Identifiant du client</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Date du commande</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Etat</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="pushButtonRechercher_2">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>30</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Rechercher</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonModifier_2">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>380</y>
         <width>141</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier </string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonSupprimer_2">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>380</y>
         <width>141</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer </string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxExcel_2">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>80</y>
         <width>83</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>EXCEL</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxPDF_2">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>50</y>
         <width>83</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonExport_2">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>20</y>
         <width>70</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>19</x>
         <y>30</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxAjouterC_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>401</width>
        <height>401</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter commande</string>
      </property>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Identifiant du commande :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditIDCOMMANDE">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>81</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Réfference du commande :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditRefferenceCommande">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>90</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>131</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Quantité d'articles :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditQuantite">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>140</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Identifiant du client :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditID">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>190</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Date du commande :</string>
       </property>
      </widget>
      <widget class="QComboBox" name="DateEditCommande_2">
       <property name="geometry">
        <rect>
         <x>202</x>
         <y>240</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>290</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Etat :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditEtat">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>290</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAjouter_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>350</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonAnnuler_2">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>350</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButtonRetour_2">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>450</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonMenuPrincipal_2">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>450</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Menu Principal</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Gestion de ventes</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1229</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
